### PR TITLE
Add angular-pouchdb

### PR DESCRIPTION
**Pull request by tlvince**

### CODE REVIEW

**Comment by jimaek** [comment]
Please rename the directory to `angular.pouchdb`

File: files/angular-pouchdb/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-pouchdb",
+  "repo": "angular-pouchdb/bower-angular-pouchdb",
+  "files": {
+    "include": ["angular-pouchdb.js", "angular-pouchdb.min.js"]
```
Comment (jimaek) [inline files/angular-pouchdb/update.json]: `angular-pouchdb.js` is not needed. The min file is enough.

Comment (jimaek) [inline files/angular-pouchdb/update.json]: Also `angular-pouchdb.min.js` doesnt exist here https://github.com/angular-pouchdb/angular-pouchdb/tree/v3.0.0
The bot will fail

Comment (tlvince) [inline files/angular-pouchdb/update.json]: It's here: https://github.com/angular-pouchdb/bower-angular-pouchdb/tree/v3.0.0. Should the `github` property in `info.ini` point to bower-angular-pouchdb?

Comment (jimaek) [inline files/angular-pouchdb/update.json]: Sorry I didnt notice that the repos are different. Then its fine, the bot will work.

**Comment by tlvince** [comment]
Do you need these changes squashed?

**Comment by jimaek** [comment]
If you can then sure, if not then our bot can do it.

**Comment by tlvince** [comment]
Since you can't rename directories via GitHub's UI and cloning the repo would've taken too long, it was quicker to open a new PR: https://github.com/jsdelivr/jsdelivr/pull/6828

**Pull request closed without merging**